pyocd/probe/tcp_client_probe.py

```diff
@@ -197,7 +197,7 @@ def _create_exception_from_status_code(self, status, message):
     _PROPERTY_CONVERTERS = {
             'capabilities':                 lambda value: [DebugProbe.Capability[v] for v in value],
             'supported_wire_protocols':     lambda value: [DebugProbe.Protocol[v] for v in value],
-            'wire_protocol':                lambda value: DebugProbe.Protocol[value],
+            'wire_protocol':                lambda value: DebugProbe.Protocol[value] if (value is not None) else None,
```

Comment:
ah, I had tried just not calling the function if value was None. This makes more sense.